.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!-- Describe your changes in detail -->
+
+## Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Types of Changes
+<!-- Please select all items that apply either now or after creating the pull request: -->
+- [ ] Documentation update
+- [ ] Bug fix (a change that fixes an issue)
```

Comment:
If you mean that all bug fixes must have an issue created first, then leave this language or change it to "resolves an issue."

If you don't think that an issue is required for all bug fixes, I would say "(a change that corrects an error)" or similar.

Reply:
I didn't really think about "issues" in the GitHub sense at all here. I'll just remove those descriptions.